src/background.ts

```diff
@@ -0,0 +1,103 @@
+"use strict";
```

Comment:
Can we put this in `src/electron/background.ts` instead?

---

Also `use strict` usage is deprecated.

Reply:
Yeah I'll look into it. `background.ts` is primarily auto generated.